src/main/java/org/opensearch/security/ssl/rest/SecuritySSLReloadCertsAction.java

```diff
@@ -82,7 +82,7 @@ public SecuritySSLReloadCertsAction(final Settings settings,
      * PUT _opendistro/_security/api/ssl/http/reloadcerts
      *
      * NOTE: No request body is required. We will assume new certificates are loaded in the paths specified in your opensearch.yml file
-     * (https://opendistro.github.io/for-elasticsearch-docs/docs/security-configuration/tls/)
+     * (https://docs-beta.opensearch.org/docs/security-configuration/tls/)
```

Comment:
Looks like the link is invalid. 
Can you verify what the right link is?
Seems like it could be: https://docs-beta.opensearch.org/docs/security/configuration/tls/

Reply:
When I copy the [link](https://docs-beta.opensearch.org/docs/security-configuration/tls/) and post in brower, it does redirect to the [correct url](https://docs-beta.opensearch.org/docs/security/configuration/tls/). But yes, I'll change it.